README

```diff
@@ -107,14 +107,14 @@ lib/include directory. There are also test programs in the test/
 directory which give examples of use.
 
 Python bindings are in bindings/python and are commented in a similar
-manner. There is also test code in the test/ directory which give
+manner. There is also a test code in the test/ directory which gives
```

Comment:
I'm 90% sure the 'give' is right too, but I might need to consult someone with better grammar skills than myself to be sure